tests/test_meshtastic_utils_message_paths.py

```
@@ -481,3 +563,503 @@ def test_on_meshtastic_message_non_text_plugin_exception(reset_meshtastic_global
+    _set_globals(config)
+    packet = _base_packet()
+    packet["decoded"]["portnum"] = "RANGE_TEST_APP"
+    packet["decoded"]["text"] = "range test payload"
+    packet.pop("channel", None)
+
+    plugin = MagicMock()
+    plugin.plugin_name = "observer"
+    plugin.handle_meshtastic_message.return_value = False
+
+    with _patch_message_deps(plugins=[plugin], patch_logger=False) as (
+        mock_logger,
```

Ruff (0.15.9)

[warning] Unpacked variable `mock_logger` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)